### PR TITLE
DT-88-fix-calendar-and-gender-dark-mode

### PR DESCRIPTION
**Pull request by Giang-st1-00**
dark mode gender, calender and change title list card, code  dropdown, add canSeePassword in 
login. Please review my code @DuyTinh2003  @lamxuanthinh

### CODE REVIEW

File: Views/SignUp/index.tsx [hunk 1]
```diff
@@ -78,6 +69,8 @@ export default function SignUp() {
             gender: data.gender,
         };
 
+        console.log(payload);
```
Comment (Duy-Tinh-Dev) [inline Views/SignUp/index.tsx]: delete log

File: components/common/Dropdown/index.tsx [hunk 2]
```diff
@@ -1,78 +1,73 @@
...
-    const handleMenu = () => {
-        setMenuIsOpen(!menuIsOpen);
+    const handleDropdown = () => {
+        setIsDropdown(!isDropdown);
     };
 
-    const handleValue = (value: any) => {
-        setValueSelected(value.value);
-        setValue && setValue("classify", value.value);
+    const handleItemSelected = (item: any) => {
```
Comment (Duy-Tinh-Dev) [inline components/common/Dropdown/index.tsx]: no any

File: components/common/Dropdown/index.tsx [hunk 3]
```diff
@@ -1,78 +1,73 @@
...
-            defaultValue={{ value: options[0].value, label: options[0].label }}
-            onChange={(value) => handleValue(value)}
-            options={options}
-            menuIsOpen={menuIsOpen}
-            onMenuOpen={handleMenu}
-            onMenuClose={handleMenu}
-            components={{
-                IndicatorSeparator: null,
+    useEffect(() => {
+        function handleClickOutside(event: any) {
```
Comment (Duy-Tinh-Dev) [inline components/common/Dropdown/index.tsx]: any

File: components/common/Dropdown/index.tsx [hunk 4]
```diff
@@ -1,78 +1,73 @@
-import { useState } from "react";
-import { IconType } from "react-icons";
-import Select from "react-select";
-
+import { IGenderOption, IOptionDropdown } from "@/model";
+import { useEffect, useRef, useState } from "react";
 interface IDropdownProps {
```
Comment (Duy-Tinh-Dev) [inline components/common/Dropdown/index.tsx:4]: enter

File: components/common/Dropdown/index.tsx [hunk 5]
```diff
@@ -1,78 +1,73 @@
...
+                className={`select-none hover:cursor-pointer bg-white dark:bg-[#1a1a1a] flex justify-between items-center px-[20px] sm:mr-0 h-[48px] rounded-xl text-[14px] border-[#52525233] dark:border-[#fff] border-2 ${classDropdown}`}
+                onClick={handleDropdown}
+            >
+                {children}
+            </div>
+            {isDropdown && (
+                <ul
+                    className={`bg-white dark:bg-[#2C2C2C] flex flex-col gap-2 absolute top-[55px] left-0 w-full  shadow-menu z-50 rounded-[10px] py-[10px] px-[20px] max-h-[60vh] overflow-y-scroll no-scrollbar  ${classOptions}`}
+                >
+                    {menuItems?.map((item: IOptionDropdown, index: number) => {
```
Comment (Duy-Tinh-Dev) [inline components/common/Dropdown/index.tsx]: ((item,index) not not necessary (item: IOptionDropdown, index: number)

File: components/common/SlideSignIn/index.tsx [hunk 6]
```diff
@@ -3,64 +3,68 @@ import Image from "next/image";
...
-                                                  {itemSlide.title}
-                                             </h1>
-                                             <p className="text-white text-[16px] font-bold text-center">
-                                                  {itemSlide.subtitle}
-                                             </p>
-                                        </div>
-                                   </div>
-                              </div>
-                         </div>
+    const [currentSlide, setCurrentSlide] = useState(dataSlideSignIn[0].key);
```
Comment (Duy-Tinh-Dev) [inline components/common/SlideSignIn/index.tsx]: enter

File: styles/common/override.css [hunk 7]
```diff
@@ -26,18 +30,40 @@
     border-radius: 30px;
 }
 
+.dark .rdtPicker .rdtDays .rdtPrev:hover,
+.dark .rdtPicker .rdtDays .rdtSwitch:hover,
+.dark .rdtPicker .rdtDays .rdtNext:hover {
+    background-color: #7c7c7c49 !important;
```
Comment (Duy-Tinh-Dev) [inline styles/common/override.css]: dont using important

**Comment by Duy-Tinh-Dev** [comment]
image
image

**Comment by Duy-Tinh-Dev** [comment]
oke

**Comment by lamxuanthinh** [comment]
Ok ! Good @Giang-st1-00 💯

**Pull request merged**